var/spack/repos/builtin/packages/xorg-proto/package.py

```diff
@@ -0,0 +1,115 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class XorgProto(MesonPackage, AutotoolsPackage, XorgPackage):
+    """The X Window System Unified Protocol provides the headers and
+    specification documents defining the core protocol the core protocol
+    and (many) extensions for the X Window System. The extensions are
+    those common among servers descended from X11R7. It also includes
+    a number of headers that aren't purely protocol related, but are
+    depended upon by many other X Window System packages to provide
+    common definitions and porting layer."""
+
+    homepage = "https://gitlab.freedesktop.org/xorg/proto/xorgproto"
+    xorg_mirror_path = "proto/xorgproto-2022.2.tar.gz"
+
+    maintainers("adamjstewart", "wdconinc")
```

Comment:
I don't want to maintain this, I have enough on my plate already 😆

Reply:
Definitely understand :+1: 

```suggestion
    maintainers("wdconinc")
```